2024/FOSDEM/osf_on_amd_5th.md

```diff
@@ -0,0 +1,287 @@
+  [amd_blobs](https://github.com/coreboot/amd_blobs/tree/master/cezanne)
+  repository in September 2022
+    * [mb/starlabs/cezanne: Add Cezanne Byte Mk
+      I](https://review.coreboot.org/c/coreboot/+/69404/)
+    * [mb/starlabs/cezanne: Add Cezanne StarBook Mk VI
+      variant](https://review.coreboot.org/c/coreboot/+/68338/)
+    * **NO UPDATE TO THE PATCHES**
+- AMD Mendocino and Phoenix still in development with the former being in more
+  advanced state, FSP not published yet
+    * **FSP published for Mendocino, but not for Phoenix**
+- The FSP publication interval is quite long (1.5 a year between Picasso FSP
+  and Cezanne FSP release to public, and 1.25 a year after Cezanne APU release
+  FSP has been published)
+    * **Interval between Cezanne and Mendocino is only 5 months**
+    * **Mendocino is Zen2 while Cezanne is Zen3, so maybe not so big update**
+    * **Mendocino has been released to the market at the end of 2022**
+
+???
+
+Cezanne is newer core architecture, but Mendocino is a newer processor in
+terms of release date.
+
+---
+
+# AMD coreboot status - now
+
+- [patches covering KGPE-D16 bootblock support are out
+  there](https://review.coreboot.org/q/topic:kgpe-d16-bootblock)
+    * they have been abandoned because of lack of activity
+    * KGPE-D16 needs some love and attention, which, unfortunately, 3mdeb
+      can't humbly provide right now without any support from community
+- Marty Plummer ("hanetzer") is working on adapting AMD FSP for
+  Picasso/Cezanne to work on a non-Chromebook device ASRock x370 Killer SLI
+    * Join [Dasharo Matrix Space](https://matrix.to/#/#dasharo:matrix.org) or
+      [Dasharo vPubs](https://vpub.dasharo.com/o/1) to know more
+
+.center[.image-20[![](/img/ryzen.jpg)]]
+
+.center.footnote[
+[Ryzen photo](https://www.flickr.com/photos/130561288@N04/51371399197)
+by Fritzchens Fritz, CC0 1.0 Universal Public Domain Dedication
+]
+
+???
+
+hanetzer is putting some serous efforts here using SerialICE to trace the
+hardware accesses in FSP and PSP. Observes hangs around accesses to PSP and
+SMU.
+
+- PSP binaries: Picasso's work, but Cezanne (and maybe others) do not work
+  with anything but soldered memory and ChromeOS type devices
+- FSP binaries: hardcoded CPUID checks for the mobile variants, which need
+  patching or a wholly different compilation parameters.
+
+---
+
+# AMD server status - last year vs now
+
+.center[.image-30[![](/img/EpycProcessor.jpg)]]
+
+.center.footnote[
+[AMD EPYC photo](https://upload.wikimedia.org/wikipedia/commons/7/7d/EpycProcessor.jpg)
+by Raysonho @ Open Grid Scheduler / Grid Engine, CC0, via Wikimedia Commons
+]
+
+.left-column50[
+
+### Early 2023
+
+- Some new initiative from Oxide on OSF for AMD Milan (EPYC 7002 series)
+  server platform presented at [OSFC
+  2022](https://www.osfc.io/2022/talks/i-have-come-to-bury-the-bios-not-to-open-it-the-need-for-holistic-systems/)
+- Nothing new on official OSF support on servers from AMD
```

Comment:
I guess things are in their repos

Reply:
Yes, this column is for comparison against the last year's presentation. I have links to repos in next slides

Reply:
This column has been removed https://github.com/3mdeb/conferences/pull/20/commits/71382d62a8fcec201d0db24cc3ab111f4d499127#diff-a36529ceeba71601b1edbcceb8639bd8b743d579e8321eff59d469b53e6ad1d0L174-L177